rollbar/__init__.py

```diff
@@ -252,7 +252,7 @@ def _get_pylons_request():
         'safe_repr': True,
         'scrub_varargs': True,
         'sizes': DEFAULT_LOCALS_SIZES,
-        'whitelisted_types': []
+        'safelisted_types': []
```

Comment:
This looks publicly accessible. => "configure by calling init() or overriding directly"

Any concern about breaking existing users?

Reply:
👍 I'd expect this change (and the ones in other repos) to be done in a backwards-compatible way. We can deprecate the old param names and phase them out under our deprecation policy (which we can write now if we don't already have one).